.travis.yml

```diff
@@ -5,37 +5,39 @@ services:
   - docker
 
 env:
-  - ARCH=x86_64 DOCKER_IMAGE=library/centos:6.8
-  - ARCH=i686 DOCKER_IMAGE=toopher/centos-i386:centos6
+  - ARCH=x86_64 DOCKER_IMAGE=quay.io/appimage/appimagebuild
```

Comment:
What is quay.io and how can we be sure it doesn't disappear tomorrow, leaving us without a way to build? Can we move those to the Docker Registry? How are these built?

Reply:
Quay.io is a public container registry created and maintained by the folks of CoreOS. I doubt it'll be unavailable in the next 3+ years.

The containers are defined by https://github.com/TheAssassin/AppImageBuild, which I'd like to move to the AppImage organisation asap.

Reply:
OK. Sounds acceptable, but what is wrong with uploading the containers to Docker Hub?

Reply:
I reviewed both services with the standard parameters I apply to all services I think about registering with and using, and I didn't like Docker Hub because of a few passages in their ToS and other legal documents.

Quay.io works great, as far as I can tell, isn't as crowded as Docker Hub, there hasn't been any downtime for the entire time I used it. Also, CoreOS as a project seems nice.